libkbfs/disk_block_cache_test.go

```diff
@@ -758,3 +760,51 @@ func TestDiskBlockCacheMoveBlock(t *testing.T) {
 	require.Equal(t, 1, cache.syncCache.numBlocks)
 	require.Equal(t, TriggeredPrefetch, prefetchStatus)
 }
+
+func TestDiskBlockCacheHomeDirPriorities(t *testing.T) {
+	t.Parallel()
+	t.Log("Test that blocks from a home directory aren't evicted when there" +
+		" are other options.")
+	cache, config := initDiskBlockCacheTest(t)
+	defer shutdownDiskBlockCacheTest(cache)
+
+	ctx := context.Background()
+
+	t.Log("Set home directories on the cache")
+	homeTLF := tlf.FakeID(11, tlf.Private)
+	cache.AddHomeTLF(ctx, homeTLF)
+	homePublicTLF := tlf.FakeID(11, tlf.Public)
+	cache.AddHomeTLF(ctx, homePublicTLF)
+
+	t.Log("Seed the cache with blocks")
+	numTlfs := 12
+	numBlocksPerTlf := 5
+	numBlocks := numTlfs * numBlocksPerTlf * 2
+	seedDiskBlockCacheForTest(t, ctx, cache, config, numTlfs,
+		numBlocksPerTlf, tlf.Private)
+	seedDiskBlockCacheForTest(t, ctx, cache, config, numTlfs,
+		numBlocksPerTlf, tlf.Public)
+
+	countToEvict := numBlocks - numBlocksPerTlf*2 - 2
+	t.Log("Evict most of the blocks in the cache")
```

Comment:
This is confusing. IMO the home TLFs should also be seeded with `numBlocksPerTlf` blocks, then this can be simpler.

Reply:
They are seeded with that many blocks, but I should add comments making that clear.